lib/skate_web/controllers/detours_controller.ex

```diff
@@ -81,6 +81,7 @@ defmodule SkateWeb.DetoursController do
                route_segments: route_segments,
                detour_shape: ors_result
              }),
+           false <- missed_stops == [],
```

Comment:
Huh, I'm actually somewhat surprised that a `TripModification` can't have 0 missed stops, because IMO it'd still be useful to at least update the shape, even if there are no missed stops. I think for now this is a good choice though.

What do you think about a todo for this or something? maybe something like this?
```suggestion
           # TODO: Revisit the idea of `TripModification`s without missed stops
           false <- missed_stops == [],
```

Reply:
Ohhhhh hmmmm. I didn't consider the need to still preserve the trip mod for turn-by-turn reasons, even if no stops are affected. But that makes a lot of sense. I think I'll even make a ticket for it, just so we have some extra tracking for this weirdness that will need product thinking